addons/account/views/report_payment_receipt_templates.xml

```diff
@@ -46,12 +46,14 @@
                         <tr t-foreach="o.move_id._get_reconciled_invoices_partials()" t-as="rec">
                             <t t-set="amount" t-value="rec[1]"/>
                             <t t-set="inv" t-value="rec[2].move_id"/>
-                            <td><span t-field="inv.invoice_date"/></td>
-                            <td><span t-field="inv.name"/></td>
-                            <td><span t-field="inv.ref"/></td>
-                            <td class="text-right"><span t-field="inv.amount_total"/></td>
-                            <td class="text-right"><span t-esc="amount" t-options="{'widget': 'monetary', 'display_currency': inv.currency_id}"/></td>
-                            <td class="text-right"><span t-field="inv.amount_residual"/></td>
```

Comment:
@simongoffin Imo, the fix should be to don't use `invoice_date`, `amount_total`, `amount_residual` and to get the amounts directly from the partials. I think the purpose of this part is to show everything that is paid by this payment, being an invoice or not. To be discussed with @tsb-odoo